src/diagnostics/LogPointsDebugAdapter.ts

```diff
@@ -81,7 +91,7 @@ export class LogPointsDebugAdapter extends LoggingDebugSession {
         if (command === 'loadSource') {
             const sourceId: string = args;
             const request: ILoadSourceRequest = { sessionId: this._sessionId, debugId: this._debugId, sourceId };
-            logPointsDebuggerClient.loadSource(null, null, null, request).then(
+            logPointsDebuggerClient.loadSource(this._siteName, this._affinityValue, this.getPublishCrednetial(), request).then(
```

Comment:
`this.getPublishCrednetial()` is mis-spelled.

Reply:
Fixed.